shopify/assets/logs/shopify.yaml

```diff
@@ -0,0 +1,246 @@
+        - type: attribute-remapper
+          name: Map `customer.email` to `usr.email`
+          enabled: true
+          sources:
+            - customer.email
+          sourceType: attribute
+          target: usr.email
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: string-builder-processor
+          name: Define `usr.name` equal to %{customer.first_name} %{customer.last_name}
+          enabled: true
+          template: "%{customer.first_name} %{customer.last_name}"
+          target: usr.name
+          replaceMissing: false
+        - type: geo-ip-parser
+          name: GeoIP Parser for `network.client.ip`
+          enabled: true
+          sources:
+            - network.client.ip
+          target: network.client.geoip
+          ip_processing_behavior: do-nothing
+        - type: user-agent-parser
+          name: Extract details from `http.useragent`
+          enabled: true
+          sources:
+            - http.useragent
+          target: http.useragent_details
+          encoded: false
+          combineVersionDetails: false
+    - type: pipeline
+      name: Customer
+      enabled: true
+      filter:
+        query: "service:customer"
+      processors:
+        - type: date-remapper
+          name: Define `updatedAt` as the official date of the log
+          enabled: true
+          sources:
+            - updatedAt
+        - type: attribute-remapper
+          name: Map `legacyResourceId` to `usr.id`
+          enabled: true
+          sources:
+            - legacyResourceId
+          sourceType: attribute
+          target: usr.id
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `displayName` to `usr.name`
+          enabled: true
+          sources:
+            - displayName
+          sourceType: attribute
+          target: usr.name
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+    - type: pipeline
+      name: Product
+      enabled: true
+      filter:
+        query: "service:product"
+      processors:
+        - type: date-remapper
+          name: Define `updatedAt` as the official date of the log
+          enabled: true
+          sources:
+            - updatedAt
+        - type: attribute-remapper
+          name: Map `status` to `product_status`
+          enabled: true
+          sources:
+            - status
+          sourceType: attribute
+          target: product_status
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - name: Lookup for `product_status` to `status`
+          enabled: true
+          source: product_status
+          target: status
+          lookupTable: |-
+            ACTIVE,info
+            ARCHIVED,info
+            DRAFT,info
+          defaultLookup: info
```

Comment:
Since everything is currently mapped to `info`, is this processor added in case there is something that makes sense to surface as a Warning or Error in the future?

Reply:
yes